package.json

```diff
@@ -19,7 +19,8 @@
     "node": ">= 6.4.0"
   },
   "scripts": {
-    "test": "check-build"
+    "jest": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Ideally we want to have a `test:unit` script which runs all the tests. So that `npm run test:unit` would run all unit tests. Remember to include the jest configuration file as well. https://jestjs.io/docs/configuration